module/zfs/vdev.c

```diff
@@ -6052,6 +6054,15 @@ vdev_prop_set(vdev_t *vd, nvlist_t *innvl, nvlist_t *outnvl)
 			}
 			vd->vdev_slow_io_t = intval;
 			break;
+		case VDEV_PROP_QUEUE_IO:
+			if (nvpair_value_uint64(elem, &intval) != 0) {
+				error = EINVAL;
+				break;
+			}
+			if (vd->vdev_ops->vdev_op_leaf) {
+				vd->vdev_queue_io = intval & 1;
```

Comment:
Do you plan it to be a bitmask, or why else ` & 1`?

Reply:
Ah, no. I removed the `& 1`.